azure-kusto-data/azure/kusto/data/client_base.py

```diff
@@ -128,9 +128,9 @@ class ExecuteRequestParams:
     def __init__(
```

Comment:
This is getting pretty messy.
Maybe split it to multiple ctors, each with the params they need:
from_query, from_mgmt, from_streaming_ingest_reader, from_streaming_ingest_uri or something